packages/dx-react-chart-demos/src/demo-sources/point-chart/scatter.jsxt

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  Chart,
+  ScatterSeries,
+  ArgumentAxis,
+  ValueAxis,
+  Grid,
+} from '@devexpress/dx-react-chart-<%&themeName%>';
+import { Scale } from '@devexpress/dx-react-chart';
+
+const getRandom = () => {
```

Comment:
Can we extract it? I've already seen the same code in another demo. The same about the `normalDistribution` and `dataGenerator ` ones. I think it can be a part of a demo data generator or something like this. Look how it implemented in [grid demos](https://github.com/DevExpress/devextreme-reactive/tree/master/packages/dx-react-grid-demos/src/demo-data).